persistence/src/main/java/com/bionicuniversity/edu/fashiontips/entity/OutboxEmail.java

```diff
@@ -0,0 +1,76 @@
+package com.bionicuniversity.edu.fashiontips.entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * Stores outbox messages
+ * Created by Sergiy on 2/4/2016.
+ */
+@Entity
+@Table(name = "outbox_emails")
+public class OutboxEmail extends BaseEntity<Long> {
```

Comment:
@barik111 You will use this entity within the collection in service impl class, thus you should override equals() and hashcode() methods. I suggest use of Objects methods (Java 7).